embed/serve.go

```diff
@@ -286,6 +286,7 @@ func (sctx *serveCtx) createMux(gwmux *gw.ServeMux, handler http.Handler) *http.
 						return outgoing
 					},
 				),
+				wsproxy.WithMaxRespBodyBufferSize(0x7fffffff),
```

Comment:
I'm scarried by unlimited buffers. 

Should this one by in the same order of magnitude as: 
https://github.com/etcd-io/etcd/blob/2c8de24afc0e888a50b86f015384360a083db701/server/embed/config.go#L180 

?

Reply:
I think messages that are generated by the server itself shouldn't be limited by the websocket library. Because it's very funny if it generates the message but is then unable to actually send it. If it wants to limit message size it should split the message itself, and that's in fact what it actually does sometimes. So I think this buffer shouldn't be limited...

Reply:
The PR to the master branch is already merged long ago

Reply:
Thank you. It was not clear from the RP description that its a backport.